### DISCUSSION TITLE
Testing the mail connection

### DISCUSSION THREAD

**Discussion by tjm-git** [mikeberger/borg_calendar#194] [Q&A]
Hello.  Is there a way for force a quick check for the mail delivery setup without waiting for something to happen?

One other issue.  Downloaded the 1.10.x .deb and installed but still shows v1.8.x.  Is this correct?

thanks,
TM

**Comment by mikeberger** [comment]
The Action menu has an option "Send Email Reminder", which will send an email reminder if there are events for tomorrow or if there are overdue todos.

The Appointment editor window has a "Send Email Reminder" button on the bottom right which will send an email with that day's events, if any.

The latest linux installer is https://github.com/mikeberger/borg_calendar/releases/download/v1.10.03/borgcalendar_1.10.03-1_amd64.deb.

It installs version 1.10.03 when I try it. It should definitely not be version 1.8.x. That version is 6 years old and had a completely different installer.  - you had to run a jar file to install that. No deb installer ever contained a version 1.8.x.

**Comment by tjm-git** [comment]
Thanks for the reply.

I was able to get an attempt at an email reminder but evidently I'm not entering 
the SMTP params correctly.  Tried several entries but I can't get it to work. 
I'll take some pointers if you have any.  After years of using Borg this is the 
first time I've tried to config the email stuff.

Ran the .deb installer again from the link in this email after removing the /opt 
directories and it says it's installing the correct version, v1.10.x, it runs 
fine but the About box still shows v1.8.2 with a build date of 1/11/16.  Any 
suggestions on this?

Thanks,
Tony

On 1/31/24 09:24, Mike Berger wrote:
> The Action menu has an option "Send Email Reminder", which will send an email 
> reminder if there are events for tomorrow or if there are overdue todos.
> 
> The Appointment editor window has a "Send Email Reminder" button on the bottom 
> right which will send an email with that day's events, if any.
> 
> The latest linux installer is 
> https://github.com/mikeberger/borg_calendar/releases/download/v1.10.03/borgcalendar_1.10.03-1_amd64.deb <https://github.com/mikeberger/borg_calendar/releases/download/v1.10.03/borgcalendar_1.10.03-1_amd64.deb>.
> 
> It installs version 1.10.03 when I try it. It should definitely not be version 
> 1.8.x. That version is 6 years old and had a completely different installer.
> 
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/mikeberger/borg_calendar/discussions/194#discussioncomment-8322759>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AK47QW6EHTS4KRKDLWSOMN3YRJV3TAVCNFSM6AAAAABCTIZT6CVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGMRSG42TS>.
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>
> 

-- 
tony mollica
***@***.***

**Comment by tjm-git** [comment]
Hello, and an update.
Got the email notices to work.  No longer a problem.

Several tries with the downloaded installer and I still show v1.8.2 and not sure 
why.  I'm running the BorgCalendar executable in the /opt/borgcalendar/bin 
folder with a date of 12/02/2022 and a size of 17320bytes.

Tony

On 1/31/24 09:24, Mike Berger wrote:
> The Action menu has an option "Send Email Reminder", which will send an email 
> reminder if there are events for tomorrow or if there are overdue todos.
> 
> The Appointment editor window has a "Send Email Reminder" button on the bottom 
> right which will send an email with that day's events, if any.
> 
> The latest linux installer is 
> https://github.com/mikeberger/borg_calendar/releases/download/v1.10.03/borgcalendar_1.10.03-1_amd64.deb <https://github.com/mikeberger/borg_calendar/releases/download/v1.10.03/borgcalendar_1.10.03-1_amd64.deb>.
> 
> It installs version 1.10.03 when I try it. It should definitely not be version 
> 1.8.x. That version is 6 years old and had a completely different installer.
> 
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/mikeberger/borg_calendar/discussions/194#discussioncomment-8322759>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AK47QW6EHTS4KRKDLWSOMN3YRJV3TAVCNFSM6AAAAABCTIZT6CVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGMRSG42TS>.
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>
> 

-- 
tony mollica
***@***.***

**Comment by mikeberger** [comment]
The new BORG cannot run the old one. The only thing that I can think of is that somehow BORG 1.8.2 is running already. Once any version of BORG is running, starting another copy of BORG will just tell the running copy to open. Kill any running version. If you see a tray icon for it then exit via the tray icon. Otherwise, do "ps -fe | grep java" and see if a borg java process is running and kill it.

Did you install BORG 1.8.2 in the past? If not, then what distro are you running?
